package.json

```diff
@@ -23,9 +23,25 @@
     "style",
     "stylesheet"
   ],
-  "main": "./lib/node-parserlib.js",
+  "main": "./src/index.js",
   "scripts": {
-    "test": "ant -f ../../build.xml test",
+    "changelog": "node scripts/changelog.js",
+    "coverage": "ytestrunner -c",
+    "pretest": "jshint src tests scripts",
+    "build": "node scripts/build.js",
+    "test-yuitest": "ytestrunner",
+    "test": "npm run test-yuitest",
```

Comment:
I think we shouldn't have `pretest` and move jshint to a separate script and use that in `test`.